lib/utils/file_utils.js

```diff
@@ -17,8 +17,7 @@
  * limitations under the License.
  */
 
-const fs = require('fs');
-const path = require('path');
+const fs = require('fs-extra');
```

Comment:
I'd rather use the `fse` name, so as not get confused between fs and fs-extra.

Reply:
See: https://github.com/jprichardson/node-fs-extra#usage

> fs-extra is a drop in replacement for native fs. All methods in fs are attached to fs-extra. All fs methods return promises if the callback isn't passed.
You don't ever need to include the original fs module again:

Reply:
we can still use a separate name if you prefer though...

Reply:
I've read that, thanks :) That's why I want the `fse` name.

Reply:
sure